src/runtime/index.ts

```
@@ -0,0 +1,847 @@
+		})
+
+	const onConsoleMessage = (
+		sessionId: string,
+		callback: (message: ConsoleMessage) => void
+	): Effect.Effect<void, never> =>
+		Effect.gen(function* () {
+			if (!consoleCallbacks.has(sessionId)) {
+				consoleCallbacks.set(sessionId, [])
+			}
+			consoleCallbacks.get(sessionId)!.push(callback)
+		})
```

Biome (2.1.2)

[error] This generator function doesn't contain yield.



(lint/correctness/useYield)